public/api-docs/v1/_build/swagger_doc.yaml

```diff
@@ -477,7 +477,7 @@ paths:
                         rejection_reason: nil
```

Comment:
Into this example, could we include what an example document change request would look like? Would be very helpful to someone trying to build an application against our API!